### PR TITLE
Saucelabs tests

### PR DESCRIPTION
**Pull request by alcapat**

### CODE REVIEW

File: rc_branch.mako [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export APACHE_BASE_PATH=/${apache_base_path}
 export API_URL=//mf-chsdi3.${deploy_target}.bgdi.ch
 export BROWSERSTACK_TARGETURL=https://mf-geoadmin3.dev.bgdi.ch/${apache_base_path}
```
Comment (loicgasser) [inline rc_branch.mako]: You can remove browserstack url I think then.

**Comment by loicgasser** [comment]
shall I add a linter for this fat PR?

**Comment by gjn** [comment]
@loicgasser Yes, linter would be great for the python part.

The BrowserstackURL needs to stay. There will be some time when we have both in parallel. ok, maybe we can adapt the tests so that both use the same url and call it E2E_TARGETURL

@alcapat Can you try to adapt?

**Comment by loicgasser** [comment]
okay will do/test.

**Comment by alcapat** [comment]
@gjn : we must remove BROWSERSTACK_TARGETURL and SAUCELABS_TARGETURL. Just use the key  E2E_TARGETURL, right ?

File: test/saucelabs/checker_test.py [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    'ch.swisstopo.lubis-bildstreifen'
+
+urls_sitemap = ['https://map.geo.admin.ch/sitemap_base.xml',
+                'https://map.geo.admin.ch/sitemap_topics.xml',
+                'https://map.geo.admin.ch/sitemap_layers.xml']
+key_words_loaderjs = [
+    'ch.swisstopo.pixelkarte-farbe',
+    'static/js/ga.js',
+    'EPSG21781.js']
+topics_list = [
```
Comment (loicgasser) [inline test/saucelabs/checker_test.py]: API_URL/rest/services

File: test/saucelabs/checker_test.py [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+    'inspire',
+    'ech']
+available_language = ['de', 'fr', 'it', 'en', 'rm']
+important_layers = [
+    'ch.swisstopo.zeitreihen',
+    'ch.swisstopo.lubis-luftbilder_schwarzweiss',
+    'ch.swisstopo.lubis-bildstreifen',
+    'ch.swisstopo.lubis-luftbilder_infrarot',
+    'ch.swisstopo.lubis-luftbilder_farbe']
+important_layers.extend(['ch.swisstopo.lubis-luftbilder-dritte-firmen',
```
Comment (loicgasser) [inline test/saucelabs/checker_test.py]: make one list

File: test/saucelabs/print_test.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
+import time
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+DEFAULT_WAIT = 10
+QUERYSTRING_KML = 'KML%7C%7Chttp:%2F%2Fopendata.utou.ch%2Furbanproto%2Fgeneva%2Fgeo%2Fkml%2FRoutes.kml'
```
Comment (loicgasser) [inline test/saucelabs/print_test.py]: use quote from urllib2 if possible and put something human readable (some humans only=)

File: test/saucelabs/search_test.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+import time
+from helpers import bCheckIfUrlHasChanged
+
+# Search testing using browserstack
+QUERYSTRING_OF_BERN = "X=200393.28&Y=596671.16"
+DEFAULT_WAIT = 15
+
+
+def wait_url_changed(driver, old_url, timeout=DEFAULT_WAIT):
```
Comment (loicgasser) [inline test/saucelabs/search_test.py]: wait function is defined multiple times, put it in helpers if possible

**Comment by alcapat** [comment]
Set same browser config for Browserstack and Saucelabs 
Win 7 : Chrome 47, 48; FFox 43, 44
Win 8.1 : Chrome 48, FFox 44
Win 10 : Chrome 48, FFox 44
IE only for Browserstack

**Comment by gjn** [comment]
@alcapat Could you squash the commits?

**Comment by alcapat** [comment]
Would you like to squash all commit ?

**Comment by gjn** [comment]
If @loicgasser wouldn't mind, yes.

**Comment by loicgasser** [comment]
yes squash it all it's fine by me.

**Comment by loicgasser** [comment]
@alcapat Can you open an issue with all the remaining issues/enhancements?

**Comment by gjn** [comment]
Thanks alot for this @alcapat !

**Pull request merged**